feat(content): Add three more human starts (#11302)
endless-sky/endless-sky: data/map planets.txt

	attributes core frontier frozen mining
	landscape land/snow17
	description `Antipode is a world of creeping glaciers and year-round snow, orbiting far enough from a dim star that even when the sun breaks through the clouds, it brings little warmth. The few settlements that exist here are built near surface deposits of rare earth metals that were discovered recently, and that are mined for use in electronics and equipment throughout this sector.`
		to display
			not "start: syndicate"
	description `	The inhabitants of Antipode are among the best downhill skiers in the galaxy, and they have made some attempts to draw off-world tourists to the ski resorts here, but the weather is so hostile that they have had very little success.`
		to display
			not "start: syndicate"

> 	description `Antipode is a frozen, unforgiving world, orbiting far enough from a dim star that even when the sun breaks through the clouds, it brings little warmth. The few settlements that exist here are built near surface deposits of rare earth metals that were discovered recently, and that are mined for use in electronics and equipment throughout this sector.`

Reviewer: > Hmmm, this is only changing a few words from the original description that doesn't feel materially different:

It seem materially different enough to me. Describing the world as "frozen" and "unforgiving" seems far more personal than saying it snows year-round, reflecting the fact that the player spent many years here.

	attributes education factory "near earth" research tourism urban
	landscape land/city20
	description `The ancestral home world of humanity, Earth has a population twice that of any other inhabited planet. Sprawling cities cover large portions of its surface, many of them overcrowded and dangerous. Some people work to scrape together enough money to leave, while at the same time others, born on distant worlds, make a pilgrimage of sorts to see this planet that once cradled the entirety of the human species.`
		to display
			not "start: syndicate"

> 	description `The ancestral home world of humanity, as well as your own birthplace, Earth has a population twice that of any other planet. Sprawling cities cover its surface, many of them overcrowded and dangerous. Some people work to scrape together enough money to leave, while at the same time others, born on distant worlds, make a pilgrimage of sorts to see this planet that once cradled the entirety of the human species.`

Reviewer 2: ```suggestion
	description `The ancestral home world of humanity, as well as your own birthplace, Earth has a population twice that of any other planet. Sprawling cities cover its surface, many of them overcrowded and dangerous. Some people work to scrape together enough money to leave, while others, born on distant worlds, make a pilgrimage of sorts to see this planet that once cradled the entirety of the human species.`
```

	attributes farming fishing hai "hai: human presence" "human tourism"
	landscape land/myrabella2
	description `Greenwater is one of the few worlds in Hai space that is home to a sizable human population. The temperature on most of the planet's surface is too warm for the Hai, but tolerable for human beings. Close to the equator are scattered human settlements for farming or fishing, which trade with the Hai cities closer to the poles for technology and equipment.`
		to display
			not "start: hai"
	
	description `Greenwater is unique among Hai worlds, not only in that it is home to a sizable human population, but also because it happens to be the world where you grew up.`
		to display
			has "start: hai"

> 	description `	Most of the humans here live as fishers and farmers on islands and archipelagoes across the equator of Greenwater, where the Hai seldom visit on account of their heavy fur coats. According to those beyond the wormhole, the climate here is like a Paradise Planet, even though it lacks the extensive terraforming common to those worlds.`

Reviewer 2: ```suggestion
	description `	Most of the humans here live as fishers and farmers on islands and archipelagoes across the equator of Greenwater, where the Hai seldom visit on account of their heavy fur coats. According to those from beyond the wormhole, the climate here is like a Paradise Planet, even though it lacks the extensive terraforming common to those worlds.`
```

		to display
			not "start: deep"
	description `	Food, fashion, and entertainment here all strike you as out of pace with broader galactic culture. The deep independent history of the planet makes it less likely to simply follow the latest Earth trends as the more recent colonies do.`
		to display
			not "start: deep"
	
	description `Midgard is a world with as much geographic diversity as Earth itself: mountains, tundra, desert, oceans, marshes, plains. It was the second world in the Deep to be settled, seven centuries ago. It is now home to dozens of major metropolitan areas and a population of over four billion, and is where you were born. Although a select few live outside the cities and work in farming or ranching, the vast majority of people here have jobs in manufacturing.`
		to display
			has "start: deep"

> 	description `	The long, independent history of the planet and region as a whole has separated its culture from the broader galaxy. An example of this is the tradition of 'tributation.' One night every year, the citizens of the Deep leave out saucers of milk and sweet bread as an offering to the 'elves:' pale, delicate beings said to protect the Deep.`

Reviewer: +1 on the suggestion. Also, these single quotes should be double quotes.

	landscape land/canyon9
	description `Muspel is a hot world of deserts, canyons, and warm-water oceans. It is mostly a mining and manufacturing planet, and as a result there are few lifelong inhabitants; most of the people here are migrant workers who are just trying to save up enough money to settle down somewhere more hospitable.`
	description `	Some tourists also come here, to walk through the sandstone canyons where the wind and rain has carved the rock into tall pillars and spires. A few particularly beautiful canyons are protected against strip mining by local law.`
	spaceport `The spaceport here is rudimentary: a collection of concrete landing pads clustered around a single large building. The building is designed with layers of color and rounded spires that mimic the canyonlands. From a distance, it would be indistinguishable from the natural rock outcroppings if it were not for the glitter of sunlight off the windows.`
	spaceport `	Inside, people move at a slower and calmer pace than you have seen elsewhere in the Deep.`
		to display
			not "start: deep"

> 	spaceport `	Inside, people move at a slower and calmer pace, most likely to appear friendlier to tourists than other parts of the Deep.`

Reviewer 3: The sentence still reads a bit weirdly to me. How about:
```suggestion
	spaceport `	Inside, people move at a slower and calmer pace, something more likely to appear friendlier to tourists than other parts of the Deep.`
```
